system/doc/design_principles/design_principles.md

```diff
@@ -42,7 +42,20 @@ represent workers:
 
 [](){: #sup6 }
 
-![Supervision Tree](assets/sup6.gif "Supervision Tree")
+```mermaid
+---
+title: Supervision Tree
+---
+flowchart
+    sup1[1] --- sup2[1] --- worker1(( ))
```

Comment:
The worker circles are very small here, maybe we can put a `w` in them so that they becomes the same size as the supervisors?

Reply:
I thought so too, but I left them empty to match the previous diagram. Maybe it would be better to puts some (longer than w) meaningful text?

Reply:
How does this look:

```mermaid
---
title: Supervision Tree
---
flowchart
    sup1[Type 1 Supervisor] --- sup2[Type 1 Supervisor] --- worker1((worker))
    sup1 --- sup1a[Type A Supervisor]

    sup1a --- sup2a[Type A Supervisor] --- worker2((worker))
    sup1a --- sup3[Type 1 Supervisor]

    sup3 --- worker3((worker))
    sup3 --- worker4((worker))
```

Reply:
Yes, much better.